toml/src/test/kotlin/org/rust/toml/crates/local/CratesLocalIndexServiceTest.kt

```diff
@@ -0,0 +1,69 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.toml.crates.local
+
+import com.intellij.testFramework.fixtures.BasePlatformTestCase
+
+class CratesLocalIndexServiceTest : BasePlatformTestCase() {
+    lateinit var cratesService: CratesLocalIndexService
```

Comment:
Since we use single global instance for testing, `lateinit var` looks redundant

Reply:
What can be the better way to handle this?

If I try to initialize the service right away then the tests fail with
```
Exception in constructor: test tokio features (java.lang.IllegalStateException: ApplicationManager.getApplication() must not be null
	at com.intellij.openapiext.UtilsKt.isUnitTestMode(utils.kt:10)
	at org.rust.toml.crates.local.CratesLocalIndexServiceImpl.<init>(CratesLocalIndexServiceImpl.kt:100)
	at org.rust.toml.crates.local.TestCratesLocalIndexServiceImpl$Companion$REAL_INSTANCE$2.invoke(TestCratesLocalIndexServiceImpl.kt:20)
	at org.rust.toml.crates.local.TestCratesLocalIndexServiceImpl$Companion$REAL_INSTANCE$2.invoke(TestCratesLocalIndexServiceImpl.kt:18)
	at kotlin.SynchronizedLazyImpl.getValue(LazyJVM.kt:74)
	at org.rust.toml.crates.local.TestCratesLocalIndexServiceImpl$Companion.getREAL_INSTANCE(TestCratesLocalIndexServiceImpl.kt)
	at org.rust.toml.crates.local.TestCratesLocalIndexServiceImpl$Companion.getRealInstance(TestCratesLocalIndexServiceImpl.kt:30)
	at org.rust.toml.crates.local.CratesLocalIndexServiceTest.<init>(CratesLocalIndexServiceTest.kt:11)
```

It happens when the service tries to check for `isUnitTestMode` in the `init` block